CHANGELOG.md

```diff
@@ -16,6 +16,10 @@
 * [ENHANCEMENT] Ingester: added new metric `cortex_ingester_active_series` to track active series more accurately. Also added options to control whether active series tracking is enabled (`-ingester.active-series-enabled`, defaults to false), and how often this metric is updated (`-ingester.active-series-update-period`) and max idle time for series to be considered inactive (`-ingester.active-series-idle-timeout`). #3153
 * [ENHANCEMENT] Blocksconvert – Builder: download plan file locally before processing it. #3209
 * [ENHANCEMENT] Store-gateway: added zone-aware replication support to blocks replication in the store-gateway. #3200
+* [ENHANCEMENT] Store-gateway: exported new metrics. #3231
+  - `cortex_bucket_store_cached_series_fetch_duration_seconds`
+  - `cortex_bucket_store_cached_postings_fetch_duration_seconds`
+  - `cortex_bucket_stores_gate_queries_max`
```

Comment:
I can't see where this metric is defined.

Reply:
The `gate_queries_max` is a new metric added to the Thanos gate, which we use here:
https://github.com/cortexproject/cortex/blob/77b21c50c3ce2091abd381697790c31a0675b69e/pkg/storegateway/bucket_stores.go#L71-L72